src/devices/sound/s_dsp.cpp

```diff
@@ -696,7 +696,7 @@ void s_dsp_device::dsp_update( s16 *sound_ptr )
 			sound_ptr++;
 
 			if (outr > 32767)
-					*sound_ptr = 32767;
+				*sound_ptr = 32767;
```

Comment:
This core should probably use `put_int_clamp` / `std::clamp` against `s16`, everywhere, while not using `int` internal storages.
(`put_int` checks only against `max` value)